winspi/winspimodule.c

```diff
@@ -368,11 +348,9 @@ winspi_spibusinfo_chipselectlinecount(PySpiBusInfoObject *self, PyObject *args)
     PyObject* result = NULL;
     VALIDATE_SPIBUSINFO(self);
 
-    Py_BEGIN_ALLOW_THREADS;
     if (SUCCESS == get_chipselectlinecount_spibusinfo(self->ob_businfo, &val)) {
```

Comment:
yeah... I agree that sucks... are we the only two people in the world complaining about this missing feature? :-)